docs/release_notes/9_5.md

```diff
@@ -52,79 +52,57 @@ In addition to the Azure Marketplace, Rocky Linux is available for free on the A
-* **Ruby 3.3** available as a module stream
-* **PHP 8.2** available as a module stream
-* **nginx 1.24** available as a module stream
-* **MariaDB 10.11** available as a module stream
-* **PostgreSQL 16** available as a module stream
+* **Apache HTTP Server 2.4.62**
+* **Node.js 22**
+
+### Compilers and development tools
 
-The following applications have been upgraded:
+For details on the changes in this category [see the upstream link here](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/pdf/9.5_release_notes/Red_Hat_Enterprise_Linux-9-9.5_Release_Notes-en-US.pdf#%23new-features-compilers-and-development-tools)
 
-* **Git** rebased to version 2.43.0
-* **Git LFS** rebased to version 3.4.1
+Updated system toolchain components:
 
-### Containers
+* **GCC 11.5**
+* **Annobin 12.70**
 
-For details on the changes in this category [see the upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/new-features#new-features-containers).
+Updated performance tools and debuggers:
 
-Notable changes include:
+* **GDB 14.2**
+* **Valgrind 3.23.0**
+* **SystemTap 5.1**
+* **elfutils 0.191**
+* **libabigail 2.5**
 
-* The `podman build farm` command for creating multi-architecture container images is available as a Technology Preview
-* Podman now supports `containers.conf` modules to load a predetermined set of configurations
-* The Container Tools package is updated
-* Podman v4.9 RESTful API now displays data of progress when you pull or push an image to the registry
-* SQLite is now fully supported as a default database backend for podman
-* `Containerfile` now supports multi-line HereDoc instructions
-* `pasta` as a network name is now deprecated
-* The BoltDB database backend is now deprecated
-* The `container-tools:4.0` module is now deprecated
-* The Container Network Interface (CNI) network stack is deprecated and will be removed in a future release
+Updated performance monitoring tools:
 
-### Compilers and development tools
+* **PCP 6.2.2**
+* **Grafana 10.2.6**
+
+Updated compiler toolsets:
 
-For details on the changes in this category [see the upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.4_release_notes/new-features#new-features-compilers-and-development-tools)
-
-* LLVM Toolset rebased to version 17.0
-* Rust Toolset rebased to version 1.75.0
-* Go Toolset rebased to version 1.21.0
-* Clang resource directory moved (from `/usr/lib64/clang/17` to `/usr/lib/clang/17`)
-* `elfutils` rebased to version 0.190
-* `systemtap` rebased to version 5.0
-* Updated GCC Toolset 13
-* `pcp` updated to version 6.2.0
-* New `grafana-selinux` package
-* New processor microarchitectures supported in `papi`
-* New package `maven-openjdk21`
-* New package `libzip-tools`
-* `cmake` rebased to version 3.26
+* **GCC Toolset 14**
+* **LLVM Toolset 18.1.8**
+* **Rust Toolset 1.79.0**
+* **Go Toolset 1.22**
 
 ## Known issues
```

Comment:
I suggest adding a specific callout to the Installer Known Issues, referencing the blank display help:

* In the installer, the Help pages are not displayed properly. The text is there for copy/paste but is invisible

Reply:
And here too 👍🏻